### PR TITLE
(fix) Fixing the aap multimedia behave test after merge.

### PR DESCRIPTION
**Pull request by mdhaman**
This depends on  https://github.com/superdesk/superdesk-core/pull/142

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,28 +27,52 @@ cache:
...
-    - cd client && npm install && bower install && cd ..
+    - if [ "${TARGET}" = "unit" ] || [ "${TARGET}" != "${TARGET/server/}" ] || [ "${TARGET}" != "${TARGET/e2e/}" ]; then
+        cd server &&
+        pip install -r dev-requirements.txt &&
+        cd .. ;
+      fi
+    - if [ "${TARGET}" = "unit" ] || [ "${TARGET}" != "${TARGET/e2e/}" ]; then
+        cd client && npm install && cd .. ;
+      fi
+    - if [ "${TARGET}" = "unit" ] || [ "${TARGET}" != "${TARGET/e2e/}" ]; then
```
Comment (mugurrus) [inline .travis.yml]: Why repeat the if? The condition is identical to the previous one.

**Comment by mugurrus** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (mdhaman) [inline .travis.yml]: agree. took it from the original travis file.

**Comment by akintolga** [comment]
:+1:

**Pull request merged**